x-pack/legacy/plugins/index_management/public/app/components/mappings_editor/components/document_fields/field_parameters/fielddata_parameter.tsx

```diff
@@ -132,13 +100,27 @@ export const FieldDataParameter = ({ field, defaultToggleValue }: Props) => {
       >
         {({ min, max }) => {
           return (
-            <EuiFormRow label={getLabel(min, max)} fullWidth>
+            <>
               {valueType === 'percentage' ? (
                 <FielddataFrequencyFilterPercentage min={min} max={max} />
               ) : (
                 <FielddataFrequencyFilterAbsolute min={min} max={max} />
               )}
-            </EuiFormRow>
+
+              <EuiSpacer size="s" />
+
+              <EuiSwitch
```

Comment:
Can we use the `compressed` version for the switch? I think that's what Michael had in mind and I think would look nicer.